### DISCUSSION TITLE
Sign in with microsoft?

### DISCUSSION THREAD

**Discussion by Gammer0909** [ORelio/Minecraft-Console-Client#2612] [Q&A]
I have a Microsoft account, as most of you should as well, and I was wondering:
*Do I need to use my Microsoft account email to sign in?*
I don't have anything on my username, I just have a Microsoft account. How do I sign in?

**Comment by milutinke** [comment]
Yes, you need to use an email.
I suggest using the Browser login method, as it's more reliable, and the built-in login method does not work with 2FA.

**Reply by Gammer0909** [reply]
How does one use the browser method?

**Reply by milutinke** [reply]
https://mccteam.github.io/guide/configuration.html#method
